llvm/lib/Target/AMDGPU/AMDGPUAttributor.cpp

```diff
@@ -740,6 +740,16 @@ struct AAAMDSizeRangeAttribute
       if (!CallerInfo || !CallerInfo->isValidState())
         return false;
 
+      /// When the caller AA is in its initial state, the state remains valid
+      /// but awaits propagation. We skip processing in this case. Note that we
+      /// must return true since the state is still considered valid.
+      if (CallerInfo->isAtInitialState()) {
+        LLVM_DEBUG(dbgs() << '[' << getName() << "] Caller "
+                          << Caller->getName()
```

Comment:
use printAsOperand to handle anonymous functions correctly

Reply:
This is copied from existing code. I'll do a follow up patch to change them all.